src/GuzzleBundleCachePlugin.php

```diff
@@ -45,6 +47,28 @@ public function loadForClient(array $config, ContainerBuilder $container, string
             $cacheMiddlewareExpression = new Expression(sprintf('service("%s")', $cacheMiddlewareDefinitionName));
 
             $handler->addMethodCall('push', [$cacheMiddlewareExpression, 'cache']);
+
+            $eventDispatcherName       = sprintf('guzzle_cache.event_dispatcher.%s', $clientName);
+            $eventDispatcherDefinition = new Definition(EventDispatcher::class);
+            $eventDispatcherDefinition
```

Comment:
it's public by default. It's no need to write it explicitly.

Reply:
in symfony 4.0 it will private by default. so I anticipated a bit. do you prefer I change it ?